### PR TITLE
Add conveniences for common redis commands.

### PR DESCRIPTION
**Pull request by John-Connolly**

### CODE REVIEW

**Comment by tanner0101** [comment]
This will need to be re-based onto #93

**Comment by tanner0101** [comment]
@John-Connolly can you take a look at the merge conflicts?

**Comment by John-Connolly** [comment]
Yep, I should have time later this week.

File: Sources/Redis/Client/Redis+Commands.swift [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+            return value
+        }
+        return resp
+    }
+    
+    /// Sets the list element at index to value.
+    public func lSet(_ item: RedisData, at index: Int, in list: String) -> Future<Void> {
+        let resp = command("LSET", [RedisData(bulk: list),
+                                    RedisData(bulk: index.description),
+                                    item
```
Comment (tanner0101) [inline Sources/Redis/Client/Redis+Commands.swift]: xcode formatting here

**Comment by tanner0101** [comment]
One comment regarding formatting. Also looks like Swiftlin is complaining about spaces at the end of newlines. There's an option in xcode for automatically trimming whitespace.

**Comment by tanner0101** [comment]
Thanks!

**Pull request merged**